docs/source/algorithms/ILLLagrange-v1.rst

```diff
@@ -0,0 +1,94 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+
+Description
+-----------
+This algorithm reduces data from IN1 - Lagrange.
+
+It can take as input multiples files from a single monochromator scan, the associated empty cell runs, a correction file
+if needed. Reduction consists in multiplying the data by the correction factor from the correction file, and subtracting
+the empty cell from the raw data.
+
+The X-axis of the result data can be offset by the incident energy using `UseIncidentEnergy`, and be converted to wave
+number instead of the default energy using `ConvertToWaveNumber`.
+
+All the files in each field are merged together in a single curve, with very close points being merged together to avoid
```

Comment:
The current behaviour removes these potentially problematic points rather than merging them, which behaviour would be more desired (probably up to the user)?